src/sentry/static/sentry/app/views/projectInstall/platform.jsx

```diff
@@ -17,6 +17,8 @@ import space from 'app/styles/space';
 import withApi from 'app/utils/withApi';
 import withOrganization from 'app/utils/withOrganization';
 
+const PrismaTheme = require('prismjs/themes/prism-tomorrow.css').toString();
```

Comment:
I think you can just do this since this is codesplit

```suggestion
import 'prismjs/themes/prism-tomorrow.css'
```